backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/jni/RustBackend.kt

```diff
@@ -98,13 +98,35 @@ class RustBackend private constructor(
     ): JniUnifiedSpendingKey {
         return withContext(SdkDispatchers.DATABASE_IO) {
             createAccount(
+                dbDataPath = dataDbFile.absolutePath,
```

Comment:
I reordered the parameters here because the database file and the network id are really environment parameters, whereas the account name and key source etc. are the arguments to the logical function that's being executed.

Reply:
Oh, I see now. Later, I sorted the parameters alphabetically, which is what we do across the SDK. Do you want me to change it as you described?

Reply:
No, consistency is better (though alphabetical parameter lists seems like not a great convention to me.)